bundle/edu.gemini.itc.web/src/main/resources/servlets/web/ITCacqCam.html

```diff
@@ -316,65 +312,70 @@
       </td>
     </tr>
   </table>
-  <p><br>
-
+  <p>
+  &nbsp;
```

Comment:
Is it possible to avoid the use of `&nbsp;`? I know that the ITC is based on old HTML and doesn't - as far as I know - use stylesheets, but I think the use of `&nbsp;` is outdated.

@cquiroz, please correct me if I'm wrong.

Reply:
I don't have any preference here, but it looks like I may have copied and pasted this bit from the other instrument forms which all use "&nbsp;" instead of "<p><br></p>".  I'll go with whatever the pro's decide.